src/model.hpp

```diff
@@ -159,7 +159,7 @@ class model {
             // events that must be delivered at the start of the next
             // integration period at the latest.
             auto exchange = [&] () {
-                PE("stepping", "communciation");
+                PE("stepping", "communication");
```

Comment:
I realised that you must have worked around it, because I was doing the same in my own scripts! But it had to get fixed some time.